comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -161,60 +161,77 @@ func (rc rcClient) start() {
 }
 
 func (rc rcClient) mrfUpdateCallback(updates map[string]state.RawConfig, applyStateCallback func(string, state.ApplyStatus)) {
-	var failover *bool
+	// If the updates map is empty, we should unset the failover settings if they were set via RC previously
+	if len(updates) == 0 {
+		mrfFailoverMetricsSource := config.Datadog.GetSource("multi_region_failover.failover_metrics")
+		mrfFailoverLogsSource := config.Datadog.GetSource("multi_region_failover.failover_logs")
+
+		// Unset the RC-sourced failover values regardless of what they are
+		config.Datadog.UnsetForSource("multi_region_failover.failover_metrics", model.SourceRC)
+		config.Datadog.UnsetForSource("multi_region_failover.failover_logs", model.SourceRC)
+
+		// If either of the values were previously set via RC, log the current values now that we've unset them
+		if mrfFailoverMetricsSource == model.SourceRC {
+			pkglog.Infof("Falling back to `multi_region_failover.failover_metrics: %t`", config.Datadog.GetBool("multi_region_failover.failover_metrics"))
+		}
+		if mrfFailoverLogsSource == model.SourceRC {
+			pkglog.Infof("Falling back to `multi_region_failover.failover_logs: %t`", config.Datadog.GetBool("multi_region_failover.failover_logs"))
+		}
+		return
+	}
+
 	applied := false
 	for cfgPath, update := range updates {
 		mrfUpdate, err := parseMultiRegionFailoverConfig(update.Config)
 		if err != nil {
-			pkglog.Errorf("MRF update unmarshal failed: %s", err)
+			pkglog.Errorf("Multi-region Failover update unmarshal failed: %s", err)
```

Comment:
The smallest of small nits: either go with `Multi-region failover` or `Multi-Region Failover`.

Reply:
Good call. Took a pass at comments and logs and updated everything to be upper case to be consistent with your previous changes. Updated in [f40e72c](https://github.com/DataDog/datadog-agent/pull/24754/commits/f40e72cc7eeff84a2d980211787166e08681795e)